<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondTab</class>
 <widget class="QWidget" name="secondTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget2"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_name_sender">
           <property name="text">
            <string>Sender's name </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sender_name_line">
           <property name="placeholderText">
            <string>Write the name of the node who sends the packet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_name_recv">
           <property name="text">
            <string>Receiver's name :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="recv_name_line">
           <property name="placeholderText">
            <string>Write the name of the node who receives the packet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_psize">
           <property name="text">
            <string>Packet's size (value greater or equal than 1) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="psize_sb">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_delay_btw_pckts">
           <property name="text">
            <string>Delay after sending the following packets (units in abbreviated form) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="delay_btw_pckts_line">
           <property name="placeholderText">
            <string>Time with units</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_time_start">
           <property name="text">
            <string>Time start (units in abbreviated form) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="time_start_line">
           <property name="placeholderText">
            <string>Time with units</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_nb_pckts">
         <property name="text">
          <string>Number of packets to send :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nb_pckts_sb">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setPcktcarac">
         <property name="text">
          <string>Set packet's caracteristics</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="add2">
     <property name="text">
      <string>Add Packet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete2">
     <property name="text">
      <string>Delete Packet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
